exporter/alibabacloudlogserviceexporter/metricsdata_to_logservice.go

```diff
@@ -64,6 +68,7 @@ func (kv *KeyValues) Replace(key, value string) {
 
 func (kv *KeyValues) AppendMap(mapVal map[string]string) {
 	for key, value := range mapVal {
+		key = sanitize(key)
```

Comment:
It looks like you can replace these 5 lines with calling the `Append` function

Reply:
Yes, AppendMap can call Apend directly